markdown/ch2.markdown

```diff
@@ -80,6 +72,14 @@ As in Chapter 1, first we will use Pandas to make our job of reading in the data
 First we will read in our counts data as a pandas table.
 
 ```python
+# Make all plots appear inline in the Jupyter notebook and set splotting style
+%matplotlib inline
+
+import matplotlib.pyplot as plt
+plt.style.use('style/elegant.mplstyle')
```

Comment:
Do we want to filter out these lines, or somewhere in the book show readers where to find it?